drgn/helpers/linux/dmesg.py

```diff
@@ -0,0 +1,158 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+"""
+dmesg-vmcore
+-------
+
+The ``drgn.helpers.linux.dmesg`` module provides helpers for reading the
+Linux kernel log.
+"""
+
+from typing import Dict, List, NamedTuple
+
+from drgn import Object, Program, cast, sizeof
+
+__all__ = ("get_dmesg", "get_printk_records")
+
+
+LOG_CONT = 8
```

Comment:
You should get this from `prog` in `dump_dmesg_lockless` and `dump_dmesg_structured`. E.g., `LOG_CONT = prog["LOG_CONT"].value_()`.